README.md

```diff
@@ -193,124 +200,167 @@ Other acceptable location examples:
 
-It's as simple as running this command:
+Example:
+- `from amazon+france,ovh+france --limit 2` returns a probe hosted on AWS in France and a probe hosted on OVH in France.
 
-```
-docker run -d --log-driver local --network host --restart=always --name globalping-probe ghcr.io/jsdelivr/globalping-probe
-```
-And it works for both x86 and ARM architectures. 
-[Podman related instructions.](https://github.com/jsdelivr/globalping-probe#podman-alternative)
+### Things to keep in mind
 
-Notes:
-- The probe doesn't open any ports or accept any incoming connections. It can only establish a connection with our API.
-- We include regularly updated lists and databases of domains and IPs that are associated with malware or potentially dangerous content and completely ban them on the API  level
-- The tests scale to the amount of available CPU cores. Our code is very lightweight and shouldn't use too many of your resources, so in most cases we recommend running our probe as is. But if you're worried you can use `--cpuset-cpus="0-2"` to limit the number of available cores.
-- We rate-limit all users on the API level to avoid the abuse of network
-- No local network tests are allowed, only public endpoints.
+#### Probes share no UUIDs
+Our probes don't expose unique IDs that you can use to target them explicitly. Our probes don't expose unique IDs that you can use to target them explicitly. Instead, we recommend fine-tuning the probe selection by using and combining filters or specifying IDs from previous measurements, as described in the best practice section above.
 
-Read more about the [Globalping Probe](https://github.com/jsdelivr/globalping-probe) in the dedicated repo.
+This ensures that popular locations, like `from Amsterdam` or `from AWS`, are automatically load-balanced across multiple probes within the same location, preventing the overload of specific probes.
 
+#### Test result variability
+When requesting a specific number of probes, there is no guarantee that the API will respond with the exact amount.
 
-## Limits | WIP
+#### Probe availability rules
+Globalping exposes and lets you interact with probes that are currently **online**. 
 
-Our platform has multiple limits to avoid abusive behaviour and at the same time motivate people to contribute to the sustainability of our platform.
+Here's when a probe is considered online and available through our API:
+- **It's up-to-date.** A probe running an older version will be forced to auto-update before it becomes available.
+- **It passes quality control tests.** Every probe must pass a test, ensuring it's connected to a reasonably stable network without packet loss. This test is done regularly; if a probe fails, it gets taken offline.
 
-#### Global limits
+This whole process is completely automated and managed by our platform.
 
-These limits are applied per IP address regardless if an API key supplied or not.
+## Join the network – Run a virtual probe
```

Comment:
All of our probes are virtual by default, so lets just call them probes.
It's RIPE Atlas that is doing the opposite, their default is hardware probes, so they would need to use the "virtual" prefix everywhere, but not us.